input/definitions/Resource/StructureDefinition-cqf-event-recorded.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "cqf-event-recorded",
+  "extension": [{
+    "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+    "valueCode": "cqi"
```

Comment:
```suggestion
    "valueCode": "fhir-i"
```

Reply:
fhir-i causes an error as it is not in the valueset (http://hl7.org/fhir/ValueSet/hl7-work-group|5.1.0)
 The following is there and used instead.
fhir | FHIR Infrastructure | FHIR Infrastructure (http://www.hl7.org/Special/committees/fiwg/index.cfm).